pages/linux/pkgfile.md

```diff
@@ -0,0 +1,16 @@
+# pkgfile
+
+> Tool for searching files from packages in the official repositories on arch-based systems.
+> More information: <https://wiki.archlinux.org/index.php/Pkgfile>.
+
+- Synchronize the pkgfile database:
+
+`pkgfile -u`
+
+- Search for a package that owns a specific file:
+
+`pkgfile {{path/to/file}}`
+
+- List all files provided by a package:
+
+`pkgfile -l {{pagkage_name}}`
```

Comment:
We prefer long arguments when are available:
```suggestion
`pkgfile --list {{pagkage_name}}`
```

Reply:
There's also a typo.  I'll update @navarroaxel's suggestion.